Homework_2.playground/Contents.swift

```diff
@@ -55,10 +123,12 @@ stack.pop()
 stack.pop()
 stack.top()
 
-class StackStatistics: Stack {
-    private let stackOfMinElements = Stack()
+/// Container for elements organized with LIFO principle. Stores the current minimum element of the Stack.
```

Comment:
```
...Stores the current minimum element of the Stack
```

What is `Stack`? Which stack. You only described a "container"

Reply:
Done.

Reply:
> ...Stores the current minimum element of the Stack

1. Is that the only thing it can store?
2. Technically, every `Stack` stores its current minimum element.

I'd suggest:
```
Allows to get the current minimum element in O(1).
```

Reply:
Done. Thanks, good suggestion.